backend/src/game.ts

```diff
@@ -147,16 +147,38 @@ export class Game {
 		const qn = this.activeQuestion;
 		const qd = this.getQuestionData();
 
+		// generate new leaderboard
 		let leaderboard: LeaderBoard[] = [];
 		this.players.forEach(function (player: User) {
 			player.scorePlayer(qn, qd);
```

Comment:
When you try to get the results, this line throws an error because `this.getQuestionData()` on line 148 returned `undefined`. I'm guessing because there is no current question?

Reply:
This should be fixed in c5a7cff